bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBFToXToHH_UL/VBFToBulkGravitonToHH_M/VBFToBulkGravitonToHH_M_proc_card.dat

```diff
@@ -0,0 +1,12 @@
+
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+import model RS_bulk_ktilda
+define q = u c d s b u~ c~ d~ s~ b~
+define p = u c d u~ c~ d~
+generate p p > y q q / g h , ( y > h h )
```

Comment:
We observed some unhealthy behaviors when you use gauge unsafe syntaxes such as "/ g h".
Is it not possible to force ygg and hgg couplings to 0 and make the contributions negligible?
Also do you only need model parameter settings that are default in your model file's parameter card (I see LRS, ktilda, clight, ctop, cg, sw2)?

Reply:
Hello @sihyunjeon : 

> We observed some unhealthy behaviors when you use gauge unsafe syntaxes such as "/ g h".

What kinds of unhealthy behaviors have you observed? 

> Is it not possible to force ygg and hgg couplings to 0 and make the contributions negligible?

We can do that if you feel strongly about it, however we already performed test runs and validated the simulation with this definition of process/parameters: [1]. Do you think these checks are sufficient? Or rather, do you think the unhealthy behavior you have observed may still be present and not visible in these checks? 

-Abe 

[1] https://twiki.cern.ch/twiki/pub/CMS/AbrahamTishelmanCharnyHomepage/29_July_2021_Res_VBF_HH_Validation.pdf

Reply:
Hello @sihyunjeon , @agrohsje - I would like to kindly ask if you have had a chance to view my reply above.

-Abe

Reply:
Can you comment more which aspects you like to remove with /g h and why? I am not sure how you conclude from your slides that removing the photon is better.

Reply:
Hey sorry for not responding. The possible issue is that this syntax induces possibilities of violating gauge invariance so it is typically much more safer to just turn off the couplings and avoid using such syntaxes. I mean, your slides look nice and everything seems to make sense, but who knows whether there could be differences or not?